### ISSUE TITLE
fetch failed error in creating payload app with  create-payload-app

### ISSUE THREAD

**Issue by lokinderchauhan** [payloadcms/payload#9086]
### Describe the Bug

`fetch failed` error comes trying to create new pay load app with command `pnpx create-payload-app@beta <app-name>` 
tried with two workflows.  choosing blank template and website,  it gets displays error message after entering `PostgreSQL connection string` . screen shot attached   ![Image](https://github.com/user-attachments/assets/bf10b6e5-7ce7-4264-9794-e3c0079a30b1)

### Link to the code that reproduces this issue

create-payload-app@3.0.0-beta.126

### Reproduction Steps

create payload application with following steps 
1 `pnpx create-payload-app@beta <app-name>`
2.  Select a database  as PostgreSql
3.  Enter PostgreSQL connection string
it will breakdown with  `fetch failed` error 

### Which area(s) are affected? (Select all that apply)

area: core

### Environment Info

create-payload-app@3.0.0-beta.126

**Comment by darrenbutcher** [comment]
Experienced this same issue as well. Tried both templates (website and blank) and tried both SQLite and Postgres. Also switched out using pnpm for npm but still encountered `fetch failed`.

**Comment by darrenbutcher** [comment]
Just to follow up I got this working by running the command while using a VPN. @r1tsuu this could be a user DNS issue, which is why it might be hard to reproduce. @lokinderchauhan you can try this and see if it resolves.

TLDR: Since there was no reference to 'fetch failed' error in the repo, I suspected it might be an error passing from the registry. During my research, I found a similar situation with Nuxt (nuxt/nuxt#21853). One user suggested changing the DNS settings and another recommended a VPN. Since I'm also using WSL2, trying a VPN first seemed easier than reconfiguring my DNS settings. Hope this helps.

**Comment by jmikrut** [comment]
Hey @darrenbutcher I think you're exactly right. I'll convert this to a discussion so that if others come across this, they'll see your notes.

Thank you!
